addons/im_livechat/static/tests/embed/composer_patch_tests.js

```diff
@@ -0,0 +1,29 @@
+/* @odoo-module */
+
+import { startServer } from "@bus/../tests/helpers/mock_python_environment";
+
+import { loadDefaultConfig, start } from "@im_livechat/../tests/embed/helper/test_utils";
+
+import { triggerHotkey } from "@web/../tests/helpers/utils";
+import { click, contains, insertText } from "@web/../tests/utils";
+
+QUnit.module("composer patch");
+
+QUnit.test("File upload only available when authorized", async () => {
+    const pyEnv = await startServer();
+    await loadDefaultConfig();
+    const env = await start();
```

Comment:
Isn't start returning an object (not env)?

Reply:
The `start` method in the embed test suite only returns `env`. It's sufficient and easier.